jedi_vim.py

```diff
@@ -157,10 +158,21 @@ def process(self):
         if not (self._process and self._process.poll() is None):
             cmd = os.path.join(
                 os.path.dirname(os.path.abspath(__file__)), self.remote_cmd)
+            # On Windows platform, need STARTUPINFO
+            if platform.system() == 'Windows':
+                si = subprocess.STARTUPINFO()
+                si.dwFlags = subprocess.STARTF_USESHOWWINDOW;
+                si.wShowWindow = subprocess.SW_HIDE;
+                stderr = subprocess.STDOUT
```

Comment:
It redirect stderr to stdout. So, if it prints warnings or such like, it corrupt RPC protocol which is also using stdout.
Maybe, redirecting stderr to /dev/null is more safe here. (but I don't know windows have /dev/null like device.)